src/julia.h

```diff
@@ -255,6 +255,7 @@ typedef struct _jl_method_t {
 
     int32_t nargs;
     int32_t called;  // bit flags: whether each of the first 8 arguments is called
+    int32_t nospec;  // bit flags: which arguments should not be specialized
```

Comment:
how many arguments can this use before it breaks down? will there be a sane error or warning with 32 or more arguments?